skipper.env

```diff
@@ -60,6 +60,8 @@ E2E_TESTS_MODE
 LOG_FOLDER
 IPv4
 IPv6
+IS_IPV4
+IS_IPV6
```

Comment:
I actually don't like that there's a couple of ways to do the same thing
Can we conform with one convention?
Also, I don't like the addition of the ``IS`` which feels redundant

Reply:
btw, moving to all caps is a really good thing you're doing here, which might be a good convention to stick to everywhere

Reply:
I removed the latter. Let's keep `IPv4` notation and discuss whether to change it to all-caps.